bundle_gems

```diff
@@ -58,6 +63,55 @@ unless File.exist?(gem_file_lock)
 end
 logger.info "Using #{gem_file_lock}"
 
+if options[:cpio]
+  FileUtils.cp ['Gemfile', 'Gemfile.lock'], options[:outdir]
+  if File.file? 'vendor.obscpio'
+    logger.info 'Extracting vendor.obscpio...'
+    FileUtils.cp 'vendor.obscpio', options[:outdir]
+    Dir.chdir(options[:outdir]) do
+      stdout_and_stderr_str, status = Open3.capture2e(
+        'cpio -i --make-directories --format=newc < vendor.obscpio'
+      )
+      logger.info stdout_and_stderr_str
+      abort(stdout_and_stderr_str) unless status.success?
+    end
+  end
+
+  if ENV['OBS_SERVICE_BUNDLE_GEMS_MIRROR_URL']
+    Dir.chdir(options[:outdir]) do
+      mirror_url = ENV['OBS_SERVICE_BUNDLE_GEMS_MIRROR_URL']
+      stdout_and_stderr_str, status = Open3.capture2e("
+        bundle.ruby2.5 config --local mirror.http://rubygems.org #{mirror_url}")
+      logger.info stdout_and_stderr_str
+      abort(stdout_and_stderr_str) unless status.success?
+      stdout_and_stderr_str, status = Open3.capture2e(
+        "bundle.ruby2.5 config --local mirror.https://rubygems.org #{mirror_url}"
+      )
+      logger.info stdout_and_stderr_str
+      abort(stdout_and_stderr_str) unless status.success?
+    end
+  end
+
+  Dir.chdir(options[:outdir]) do
+    stdout_and_stderr_str, status = Open3.capture2e(
+      'bundle.ruby2.5 package --no-install --path vendor/bundle'
```

Comment:
ruby version hardcoded?

Reply:
argh, that was for debugging! Will remove it!